core/src/main/java/org/apache/servicecomb/core/SCBEngine.java

```diff
@@ -90,6 +90,10 @@ public static SCBEngine getInstance() {
     return INSTANCE;
   }
 
+  public EventBus getEventBus() {
```

Comment:
getEventBus has no reference.

Reply:
add SCBEngine reference to BootEvent
and then customes can get eventBus from scbEngine, no need to use the global variable directly